.github/workflows/nightly.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
+      - run: cargo install cross
```

Comment:
Can we do it this way please? https://github.com/timberio/vector/blob/648e82ca32acd1d3b08ff949983d55c3ad97f2e1/.github/workflows/tests.yml#L37-L39